qiime_studio/server.py

```diff
@@ -25,6 +33,7 @@ def hello():
 
 def start_server():
     studio.debug = True
+    studio.before_request(flush_stdio)
```

Comment:
Yeah, `after_request` seems to be working better, just have to pass the response through it.